### PR TITLE
Fix case study update

### PR DESCRIPTION
**Pull request by richtier**
[fix this](https://sentry.ci.uktrade.io/dit/directory-api-dev/issues/2229/)

### CODE REVIEW

File: company/management/commands/populate_elasticsearch.py [hunk 1]
```diff
@@ -7,7 +7,7 @@ class Command(BaseCommand):
     help = 'Populates ElasticSearch with companies and case studies'
 
     def add_arguments(self, parser):
-        parser.add_argument('--async', dest='async', type=bool, default=True)
+        parser.add_argument('--async', dest='async', type=bool, default=False)
```
Comment (konradko) [inline company/management/commands/populate_elasticsearch.py]: What's this

File: company/serializers.py [hunk 2]
```diff
@@ -64,6 +64,12 @@ class Meta:
         )
         read_only_fields = ('slug',)
 
+    def to_internal_value(self, data):
+        if isinstance(data, QueryDict):
```
Comment (konradko) [inline company/serializers.py]: what else can it be?

**Comment by konradko** [comment]
ci failed cause of the ES change:

```
Results (102.91s):
     390 passed
       1 failed
         - company/tests/test_commands.py:35 test_populate_elasticsearch
       7 skipped```

File: company/management/commands/populate_elasticsearch.py (hunk 1 above)
Comment (richtier) [reply company/management/commands/populate_elasticsearch.py]: Oops, should not have been committed

File: company/serializers.py (hunk 2 above)
Comment (richtier) [reply company/serializers.py]: Dict or QueryDict

**Pull request merged**